pkg/cli/cli.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 type kernelSettings struct {
-	KillTimeout time.Duration `cfg:"killTimeout" default:"10s"`
+	KillTimeout time.Duration `cfg:"kill_timeout" default:"10s"`
```

Comment:
Do we still need this as a separate setting here? It's already defined somewhere else, not sure where though :thinking:

Reply:
you're right, this is not needed anymore :+1: